decidim-verifications/README.md

```diff
@@ -97,6 +97,18 @@ Decidim implements two type of authorization methods:
   * `edit_authorization_path`: This is the entry point to resume an existing
     authorization process.
 
+* _Renewable authorizations_.  
+  By default a participant can't renew its authorization, but this can be enabled when registering the workflow, a time between renews can be configured (zero by default):
+
+  ```ruby
+  # config/initializers/decidim.rb
+
+  Decidim::Verifications.register_workflow(:census) do |workflow|
+    workflow.form = "myAuthorizationHandlerClass"
+    workflow.renewable = true
+    workflow.time_between_renewals = 5.minutes
```

Comment:
Lets change this to 1.day or something like that please (although it's documentation only, it'd be better to have a saner default, as this is potentially an attack area). 
For instance if I can send (by an script) an SMS every 5 minutes, and if every SMS is $ 0.05 then I can make an expense of $ 14.40 every day.

Reply:
+1 to this!

Reply:
@andreslucena changed the documentation as you suggested and also the manifests default value to 1.day as suggested by [Leo](https://github.com/decidim/decidim/pull/5854#discussion_r392123193), not 0.minutes, for this same reason 😄

Reply:
Great!